src/Console/Commands/DefaultCommand.php

```diff
@@ -54,10 +54,20 @@ protected function renderIssue(OutputInterface $output, Issue $issue, string $cu
         renderUsing($output);
 
         $file = str_replace($currentDirectory, '.', $issue->file);
+        $lineInfo = ($issue->line !== 0) ? " on line <strong>{$issue->line}</strong>" : '';
+        $suggestions = implode(', ', $issue->misspelling->suggestions);
 
         render(<<<HTML
-            <div class="text-red-700 pl-2">
-                <span>{$issue->misspelling->word} in {$file}.</span>
+            <div class="mx-2 mb-1">
+                <div class="space-x-1">
+                    <span class="bg-red text-white px-1 font-bold">ISSUE</span>
+                    <span>Misspelling detected in <strong>{$file}</strong>{$lineInfo}: '<strong>{$issue->misspelling->word}</strong>'</span>
```

Comment:
I would add an `<a>` tag to open the file here.

Reply:
@xiCO2k Great recommendation, I added this feature immediately!